src/_cffi_src/openssl/ec.py

```diff
@@ -29,7 +29,11 @@
     int nid;
     const char *comment;
 } EC_builtin_curve;
-typedef enum { ... } point_conversion_form_t;
+typedef enum {
+    POINT_CONVERSION_COMPRESSED = 2,
+    POINT_CONVERSION_UNCOMPRESSED = 4,
+    POINT_CONVERSION_HYBRID = 6
```

Comment:
Looks like we can drop the values and have cffi obtain them during the compile phase. Updating the PR.